classes/notification/managerDelegate/PendingRevisionsNotificationManager.inc.php

```diff
@@ -97,6 +97,7 @@ public function updateNotification($request, $userIds, $assocType, $assocId) {
 		$userId = current($userIds);
 		$submissionId = $assocId;
 		$stageData = $this->_getStageDataByType();
+		if ($stageData == null) return;
```

Comment:
WORKFLOW_STAGE_ID_INTERNAL_REVIEW is also in this class, because the NOTIFICATION_TYPE_PENDING_INTERNAL_REVISIONS is just in pkp-lib s. below. Thus in OJS the WORKFLOW_STAGE_ID_INTERNAL_REVIEW will not be found, creating a php notice when _getStageDataByType() below is called.